dashboard/src/components/Config/AppRepoList/AppRepoForm.scss

```diff
@@ -1,53 +1,7 @@
-.app-repo-form {
-  color: var(--cds-global-typography-color-300, #454545);
-  white-space: normal;
-  text-align: left;
-  width: 97%;
-  h2 {
-    margin-top: 0;
-  }
-  .clr-form-description {
-    font-size: smaller;
-    strong {
-      font-weight: var(--clr-forms-label-font-weight, 600);
-      color: var(--cds-global-typography-color-300, #454545);
-    }
-  }
-  .clr-form-columns {
-    width: 100%;
-    .column-valing-center {
-      height: 100%;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-    }
-    .clr-control-label-radio {
-      font-weight: normal;
-      padding: 0.2rem;
-      input {
-        margin-right: 0.2rem;
-      }
-    }
-    .clr-control-label-checkbox {
-      font-weight: normal;
-      color: var(--cds-global-typography-color-300, #454545);
-      font-size: 0.65rem;
-    }
-  }
-  .clr-form-separator-sm {
-    margin-top: 0.3rem;
-    width: 100%;
-  }
-  .clr-form-separator {
-    margin-top: 1.2rem;
-    width: 100%;
-  }
-  .clr-checkbox-wrapper input[type="checkbox"] {
-    opacity: 1;
-  }
-  .secondary-input {
-    margin-top: 0.3rem;
-    padding: 0.6rem;
-    background-color: var(--cds-global-typography-color-100, #fff);
-  }
+.cds-textarea-fix {
+  font: caption;
```

Comment:
this one is tricky. If not set, the clr-ui conflicts with the CdsTextarea causing that the textarea is not editable in Chrome (fun: https://github.com/vmware/clarity/issues/5687)